src/components/admin/PendingTable.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import Toolbar from '@material-ui/core/Toolbar';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Paper from '@material-ui/core/Paper';
+import MoreHorizIcon from '@material-ui/icons/MoreHoriz';
+import TableFooter from '@material-ui/core/TableFooter';
+import TablePagination from '@material-ui/core/TablePagination';
+import styles from './admin.module.css';
+import {Typography} from '@material-ui/core';
+
+import TablePaginationActions from './TablePaginationActions';
+
+function createData(id, name, email) {
+  return {id, name, email};
+}
+
+const rows = [
+  createData('1', 'Alice Joy', 'alicee@gmail.com'),
+  createData('2', 'David Toms', 'dt@yahoo.com'),
+];
+
+export default function PendingTable() {
+  const [page, setPage] = React.useState(0);
+  const [rowsPerPage, setRowsPerPage] = React.useState(5);
+
+  const emptyRows = rowsPerPage - Math.min(rowsPerPage,
+      rows.length - page * rowsPerPage);
+
+  const handleChangePage = (event, newPage) => {
+    setPage(newPage);
+  };
+
+  const handleChangeRowsPerPage = (event) => {
+    setRowsPerPage(parseInt(event.target.value, 10));
+    setPage(0);
+  };
+
+  return (
+    <Paper>
+      <Toolbar className={styles.titlePendingBar} variant='dense'>
+        <Typography variant='h6'>
+          Pending requests
+        </Typography>
+      </Toolbar>
+      <TableContainer>
+        <Table aria-label='Pending memberships' size='small'>
+          <TableHead>
+            <TableRow>
+              <TableCell>Name</TableCell>
+              <TableCell>Email</TableCell>
+              <TableCell></TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {(rowsPerPage > 0 ?
+            rows.slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage) :
+                rows
```

Comment:
can we move this calculation into a function? I find the ternary operator very confusing here.